docs/api-types/service-export.md

```diff
@@ -0,0 +1,39 @@
+# ServiceExport API Reference
+
+## Introduction
+
+In AWS Gateway API Controller, `ServiceExport` enables a Service for multi-cluster traffic setup.
+Clusters can import the exported service with [`ServiceImport`](service-import.md) resource.
+
+Internally, creating a ServiceExport creates a standalone VPC Lattice [target group](https://docs.aws.amazon.com/vpc-lattice/latest/ug/target-groups.html).
+Even without ServiceImports, creating ServiceExports can be useful in case you only need the target groups created;
+for example, using target groups in the VPC Lattice setup outside Kubernetes.
```

Comment:
A second thought on it, I dont think we should advertise TG creation with ServiceExports, because TG now "ephemeral" with random suffix. It will break use of TG outside of controller.

Reply:
Target groups are not ephemeral, especially when created from ServiceExports. The only exception is when users intentionally change immutable fields, but that happens regardless of random suffix anyways.